src/styles/buy.scss

```
@@ -0,0 +1,30 @@
+.buy {
+  margin: 16px;
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  border: 0;
+
+  &__link {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  &:hover {
+    background-color: #fff;
+    border: 1px solid #00acdc;
+    box-sizing: border-box;
+    border-radius: 5px;
+  }
+
+  &:hover &__link {
+    color: #00acdc;
+  }
```

```suggestion
  &:hover {
    background-color: #fff;
    border: 1px solid #00acdc;
    box-sizing: border-box;
    border-radius: 5px;
    color: #00acdc;
  }
```
color треба міняти в кнопки на хавері